imap_processing/tests/swapi/test_swapi_l1.py

```diff
@@ -49,19 +50,18 @@ def test_filter_good_data():
     bad_data_indices = filter_good_data(ds)
     assert len(bad_data_indices) == 36
 
-    # Check for bad MODE data, only HVENG is "good"
-    # TODO: update test when we update MODE from HVENG to HVSCI
+    # Check for bad MODE data, only HVSCI is "good"
     ds["mode"] = xr.DataArray(
         np.repeat(
-            [SWAPIMODE.LVENG.value, SWAPIMODE.LVSCI.value, SWAPIMODE.HVENG.value], 12
```

Comment:
Is HVENG no longer good? Why?

Reply:
Yes. Before I didn't have HVSCI data which is actually what we want after launch. I left HVENG as placeholder until today.